package.json

```diff
@@ -169,6 +169,11 @@
         "command": "go.browse.packages",
         "title": "Go: Browse Packages",
         "description": "Browse packages and Go files inside the packages."
+      },
+      {
+        "command": "go.get.import",
+        "title": "Go: Get Import on Curent Line",
```

Comment:
How about `Go: Import Package`?  `Go: Install Package` ? `Go: Get Package` ? `Go: Install Imported Package`
(naming is seriously the hardest thing to do)

And the command  would be more intuitive if it had the word "package" like `go.get.package` or `go.import.package`

Also update the description. It should be clear that we run a `go get -u` on the package

Reply:
I think `Go: Get Package` is the best one out of all of them, because it follows the semantics of using `go get`. I'll change that now.

Reply:
Agreed